scripts/import/import_all_sv_data.pl

```diff
@@ -955,7 +963,21 @@ sub get_gender {
   return $genders{$row};
 }
 
+# Get the phenotype attrib id
+# By default, attrib = 'trait'
+sub get_attrib_id {
+  my $stmt = $dbVar->prepare(qq { SELECT attrib_id FROM attrib WHERE value = 'trait' });
+  $stmt->execute() || die;
+  my $attrib_id = ($stmt->fetchrow_array)[0];
+
+  debug(localtime()." No attribute 'trait' was found in attrib table!\n") unless defined $attrib_id;
+
+  return $attrib_id;
```

Comment:
Should an undef $attrib_id be returned and the script to proceed?

Reply:
I chose to return an undef attrib_id and proceed with the import because the missing attrib_id it's not critical for the import and can be fixed afterwards. 
If I stop the import, then I'll need to do some cleaning (tmp tables) before importing again.

Reply:
It's better to kill the import and fix the attrib before. I wrote an error that also explains some cleaning has to be done before importing again.